README.md

```diff
@@ -416,6 +476,30 @@ If you do not want to use SSL set the `sslmode` to `disable`, other possible val
 [documentation](https://jdbc.postgresql.org/documentation/head/ssl-client.html). For other supported configuration
 options see [PostgresStorageConfiguration.kt](./model/src/main/kotlin/config/PostgresStorageConfiguration.kt).
 
+### ClearlyDefined Storage
+
+[ClearlyDefined](https://clearlydefined.io) is a service offering curated metadata for OpenSource components. This
+includes scan results that can be used by the scanner (if they have been generated by a compatible scanner version
```

Comment:
Let's say "ORT's _scanner_ tool" to make more clear that "scanner" is not really a concrete scanner here, but ORT's wrapper.